design/FY2021/NFP-ThreeNewSteamSystems.md

```diff
@@ -0,0 +1,170 @@
+
+These assumptions are applied to the new objects: LoadProfile:Plant in a steam loop, districtheatingsteam, and steam to water heat exchanger.
+
+## Approach ##
+
+1. The current LoadProfile:Plant calculates the outlet water temperature based on the inlet water temperature from the plant loop and user inputs for the scheduled plant load and the requested flow rate. 
+In the new LoadProfile:Plant, three additional input fields were added: Plant Loop Fluid Type (Water or steam); Degree of SubCooling (optional input for steam loop); and Degree of Loop SubCooling (otional input for steam loop). The new LoadProfile:Plant in a steam loop calculates the outlet mass flow rate based on the scheduled plant load and user inputs of degree of subcooling, because the inlet steam temperature and the outlet steam temperature before the steam trap are fixed to saturation temperature according to the assumption.
+
+2. The current DistrictHeating or DistrictCooling calculates the output capacity necessary from the inlet temperature to the setpoint temperature for that loop with the given mass flow rate in Watts. DistrictHeatingSteam calculates the output capacity necessary from the latent heat with the given saturation temperature.
+
+3. 
+Figure 1 describes the loop structure with steam to water heat exchanger.
+
+![figure1](https://github.com/EnergyPlus/blob/AddThreeSteamModulesWithNTUMethod/design/FY2021/figure1.png)
+
+
+## Testing/Validation/Data Sources ##
+
+1. Standard unit tests were used to verify the new modules.
+2. Test performance of the new modules with example test files.
+
+## Input Output Reference Documentation ##
+
+
+## Input Description ##
+
+HeatExchanger:SteamToWater,
+        \memo A steam to water heat exchanger designed to couple the steam loop to the water loop
+   A1 , \field Name
+        \required-field
+        \reference-class-name validPlantEquipmentTypes
+        \reference validPlantEquipmentNames
+        \reference-class-name validCondenserEquipmentTypes
+        \reference validCondenserEquipmentNames
+        \reference-class-name validBranchEquipmentTypes
+        \reference validBranchEquipmentNames
+   A2 , \field Availability Schedule Name
+        \note Availability schedule name for this system. Schedule value > 0 means the system is available.
+        \note If this field is blank, the system is always available.
+        \note default is that heat exchanger is on
+        \type object-list
+        \object-list ScheduleNames
+   A3 , \field Steam Inlet Node Name
+        \type node
+        \required-field
+   A4 , \field Steam Outlet Node Name
+        \type node
+        \required-field
+   N1 , \field Steam Side Design Flow Rate
+        \type real
+        \required-field
+        \minimum> 0.0
+        \units m3/s
+        \ip-units gal/min
+        \autosizable
+   A5 , \field Water Inlet Node Name
```

Comment:
I would expect that the steam connection to HX would be on the demand side of the steam loop.  And the water connection to HX would be on the supply side of a water loop.  The idea is that steam provides a source of heating for a water loop.

Reply:
@EnergyArchmage  Thanks for catching this. I did make the steam connection to HX would be on the demand side of the steam loop, and the water connection to HX would be on the supply side of a water loop. What I missed was fixing this \note in IDD, since I copied these lines from `HeatExchanger:FluidToFluid`. Thanks again for catching this. I'll fix this and I'll update the design doc by tonight.